WordPress/Resources/it.lproj/InfoPlist.strings

```diff
@@ -1,5 +1,9 @@
-NSLocationUsageDescription = "WordPress vorrebbe aggiungere la tua posizione agli articoli che pubblichi sui siti in cui hai attivato il geotagging.";
-NSLocationWhenInUseUsageDescription = "WordPress vorrebbe aggiungere la tua posizione agli articoli che pubblichi sui siti in cui hai attivato il geotagging.";
-NSCameraUsageDescription = "Per acquisire foto o video da usare nei tuoi articoli.";
-NSPhotoLibraryUsageDescription = "Per aggiungere foto o video ai tuoi articoli.";
-NSMicrophoneUsageDescription = "Per far sì che i video dispongano di audio.";
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--Warning: Auto-generated file, do not edit.-->
+<plist version="1.0">
+  <dict>
+    <key>NSLocationWhenInUseUsageDescription</key>
+    <string>FAKE NSCameraUsageDescription TRANSLATION</string>
```

Comment:
Here's the fake value I introduced in 418bcc18fe picked up by the automation and inserted in the corresponding `.strings` file.